srcpkgs/vosk-api/template

```diff
@@ -0,0 +1,78 @@
+# Template file for 'vosk-api'
+pkgname=vosk-api
+version=0.3.43
+revision=1
+_openblas_ver=0.3.20
+_clapack_ver=3.2.1
+_openfst_commit=7dfd808194105162f20084bb4d8e4ee4b65266d5
+_kaldi_commit=6417ac1dece94783e80dfbac0148604685d27579
+build_style=gnu-configure
+make_use_env=yes
+make_build_args="-C src KALDI_ROOT=../../kaldi-${_kaldi_commit}
+	OPENFST_ROOT=../../kaldi-${_kaldi_commit}/tools/openfst
+	OPENBLAS_ROOT=../../kaldi-${_kaldi_commit}/tools/OpenBLAS/install"
+hostmakedepends="autoconf automake cmake libtool"
+makedepends="blas-devel lapack-devel openblas-devel"
+short_desc="Offline speech recognition toolkit"
+maintainer="John Gebbie <me@johngebbie.com>"
+license="Apache-2.0"
+homepage="https://alphacephei.com/vosk"
+distfiles="https://github.com/alphacep/vosk-api/archive/v$version/vosk-api-$version.tar.gz
+https://github.com/alphacep/kaldi/archive/$_kaldi_commit.tar.gz
+https://github.com/alphacep/openfst/archive/$_openfst_commit.tar.gz
+https://alphacephei.com/vosk/models/vosk-model-small-en-us-0.15.zip"
+checksum="50fa82bfbfe3fc5be868237c84ac180eb55da0b65b1803bf022899673afda024
+4e5c08c9c61bbc995e0620321feb0973bb0321442f1bd8f08b47497f55efadda
+a7bac90acb9a12fdfe50c240c9bb730a7bcb5c3b3ac32017c4d27177b059680a
+30f26242c4eb449f948e42cb302dd7a686cb29a3423a8367f99ff41780942498"
+patch_args="-Np1 ../kaldi-${_kaldi_commit}/src/matrix/cblas-wrappers.h"
+
+post_extract() {
+	cd "${XBPS_BUILDDIR}/vosk-api-${version}/src"
```

Comment:
use create_wrksrc to contain all of the extracted distfiles in one directory and fix cleaning